guides/v2.2/install-gde/prereq/install-rabbitmq.md

```diff
@@ -24,8 +24,8 @@ The message queue system must be established before you install Magento. The bas
 1. Install RabbitMQ and any prerequisites.
 1. Connect RabbitMQ and Magento.
 
-{:.bs-callout .bs-callout-info}
-A basic message queue system can be implemented on {{site.data.var.ee}} using cron instead of RabbitMQ. See [Configure message queues]({{ page.baseurl }}/config-guide/mq/manage-mysql.html) for more information.
+{:.bs-callout-info}
+You can use MySQL or RabbitMQ for message queue processing. For details on setting up the message queue system, see [Message queues overview]({{ page.baseurl }}/extension-dev-guide/message-queues/message-queues.html). If you are using the Bulk API with {{ site.data.var.ee }} v2.3.3 or later, the default message queue configuration uses RabbitMQ as the message broker.
```

Comment:
I wouldn't specify a Magento version here, or at least change it to 2.3.0. I think the message queue architecture was introduced somewhere in Magento 2.2 for Magento Commerce and made available for Magento Open Source with 2.3 and it probably always already supported RabbitMQ? Bulk API itself was introduced in version 2.3.0 as far as I know.